presidio-cli/presidio_cli/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""An python CLI for analyze PII Entities with Microsoft Presidio framework."""
+
+
+APP_NAME = "presidio_cli"
+APP_VERSION = "0.0.8"
+APP_DESCRIPTION = __doc__
+SHELL_NAME = "presidio"
+
+__author__ = "Insights Engineering"
+__author_email__ = "basel.nestcicd@roche.com"
```

Comment:
@omri374 we need to check this against MS OSS.

Reply:
I didn't get a clear response for this matter. @walkowi would it be ok to remove those details? especially the copyright and license as they contradict the presidio MIT license.

Reply:
I think this can be removed. @cicdguy can you confirm?

Reply:
Ok, details removed.